lib/logging/graylog_defaults.rb

```diff
@@ -0,0 +1,30 @@
+module Logging
+  module GraylogDefaults
+    def self.config
+      [
+        ENV.fetch("GRAYLOG_HOST_NAME", "graylog.zooniverse.org"),
```

Comment:
`graylog.zooniverse.org` isn't directly accessible by Panoptes. Maybe make this `panoptes-logs.zooniverse.org` instead?  That still won't be right for staging, which uses `panoptes-staging-logs.zooniverse.org`, but at least that's one less config file we need to set it in.

Reply:
`panoptes-logs.zooniverse.org` is accessible from both staging and prod vpcs?

Reply:
Nope, staging can only access `panoptes-staging-logs.zooniverse.org` so we'll still need to set `GRAYLOG_HOST_NAME` there.